app/post/directives/views/post-view-directive.js

```diff
@@ -20,17 +20,20 @@ function (
                 $scope.currentView = 'map';
             }
 
-            if (!PostViewHelper.isViewAvailable($scope.currentView)) {
-                $scope.unavailableView = $scope.currentView;
-                $scope.currentView = 'unavailable';
-            }
+            // Watch views, in case we get new feature config
+            PostViewHelper.isViewAvailable($scope.currentView).then(function (available) {
+                if (!available) {
```

Comment:
Could this all be:

Features.loadFeatures().then(function (features) {
           if(features.isViewEnabled(view) {....

Do we want to keep the PostViewHelper or could it ultimately be replaced with an improved version of features? Or is the separation of the two better?